### PR TITLE
Remodeled db to have snake_case

### PR DESCRIPTION
**Pull request by jackofseattle**
And updated the code so that it would reference those fields properly.

### CODE REVIEW

File: server/routes/comments.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ class Comments {
     } catch(err) {
       return res
         .status(500)
-        .send('unable to complete request', err);
+        .send('unable to complete request' + err);
```
Comment (jshor) [inline server/routes/comments.js]: `err` is an object

Comment (jackofseattle) [inline server/routes/comments.js]: indeed - it works just fine like this though.  calling ```toString()``` on error objects returns the stack

File: client/vivace/list/controller.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-  '$location',
-  function(DataModel, $location) {
+  function(DataModel) {
     let promise = DataModel.query({collection: 'candidates'}).$promise;
 
     promise.then((data) => {
       this.users = data;
     }, () => {
-      $location.path('/not-found');
+      this.hasError = true;
```
Comment (jshor) [inline client/vivace/list/controller.js]: what do we do with `hasError`?

Comment (jackofseattle) [inline client/vivace/list/controller.js]: Nothing at the moment.  I will hook something up on the template eventually.  One thing's for sure though, a empty list is not the same as 'not found'

**Comment by jshor** [comment]
i don't know if it's a good idea to change the field names in the frontend to use snake case too, since it might get confusing...

**Comment by jackofseattle** [comment]
As it turns out, nothing changed on the front end. 
Further, the only fields that don't have camelCase aliases are fields that shouldn't be updated anyways - so I think we'll be ok.

**Comment by jshor** [comment]
you're right, looks like you've covered all your bases - LGTM

**Pull request merged**